apps/ledger-live-mobile/src/screens/CustomImage/Step2Preview.tsx

```diff
@@ -200,9 +188,7 @@ const Step2Preview = ({ navigation, route }: NavigationProps) => {
           size="large"
           type="main"
           outline={false}
-          onPress={requestRawResult}
-          pending={rawResultLoading}
-          displayContentWhenPending
+          onPress={handleNavigateToPreview}
```

Comment:
basically revert these change to eliminate the need to have the processing logic in PreviewPostEdit

Reply:
Done